### DISCUSSION TITLE
Self-host Site provisioning API Bearer token problem

### DISCUSSION THREAD

**Discussion by digital88** [plausible/analytics#3594] [Self-Hosted Support] [answered]
Hello.

I'm trying to use [Site provisioning API](https://plausible.io/docs/sites-api) on my self-hosted Plausible instance. I keep getting message:
`{"error":"Invalid API key. Please make sure you're using a valid API key with access to the resource you've requested."}`
But the same bearer token works for Stats API without problem. Is sites API available only for cloud version? If not, then how can I create API key/token for it?

**Comment by ruslandoga** [comment]
👋 @digital88 

The auth seems to be different for these two routes as per https://github.com/plausible/analytics/blob/0621fc0a77cc50add00baddbdd59f378324da483/lib/plausible_web/router.ex#L88-L107

**Reply by digital88** [reply, marked as answer]
Thanks!
I added missing scope in api_keys table like this:

![image](https://github.com/plausible/analytics/assets/10961828/051cb2c4-5f58-4ba0-a468-0b6ae32864cc)

After that it works perfectly.

Not sure why its not enabled in selfhosted by default?
